src/compiler/checker.ts

```diff
@@ -4766,15 +4767,19 @@ namespace ts {
                 }
                 else if (!jsDocType && isBinaryExpression(expression)) {
                     // If we don't have an explicit JSDoc type, get the type from the expression.
-                    let type = getWidenedLiteralType(checkExpressionCached(expression.right));
+                    const isModuleExportsAlias = aliased !== symbol;

```

Comment:
Might be neater to make `aliased` optional.

Reply:
I agree. Done.